docs/src/main/sphinx/sql/set-time-zone.rst

```diff
@@ -0,0 +1,66 @@
+=============
+SET TIME ZONE
+=============
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET TIME ZONE LOCAL
+    SET TIME ZONE expression
+
+Description
+-----------
+
+Sets the default time zone for the current session.
+
+If the ``LOCAL`` option is specified, the time zone for the current session
+is set to the to the default time zone of the Trino server.
```

Comment:
This is actually incorrect. It should be "the initial time zone of the session". This can be either the same as that of the Trino server or a time zone specified by the client via the X-Trino-Time-Zone header (no need to mention those details, though).

Reply:
You are right @martint . The time zone is actually set on the client (through `io.trino.cli.ClientOptions#timeZone` option).

I find the request from @mosabua  justified regarding where the default timezone is being taken from:
https://github.com/trinodb/trino/pull/8135#discussion_r651992737

Probably this explanation should be the object of a different PR.

Reply:
@mosabua I just had a look over https://trino.io/docs/current/installation/cli.html
Is there a listing available with all the options for `trino` CLI tool?